ci-operator/step-registry/rosa/aws/sts/advanced/rosa-aws-sts-advanced-workflow.yaml

```diff
@@ -22,7 +22,7 @@ workflow:
       DEFAULT_MACHINE_POOL_LABELS: "k8s.ovn.org/egress-assignable="
       MP_REPLICAS: "1"
       LOCAL_ZONE: "true"
-      LOCALZONE_WORKER_ASSIGN_PUBLIC_IP: "no"
+      EDGE_NODE_WORKER_ASSIGN_PUBLIC_IP: "no"
```

Comment:
@yasun1 please review ROSA part, thanks

Reply:
it works as expected. See [4.15-amd64-stable-aws-rosa-sts-advanced-f7](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/48418/rehearse-48418-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-aws-rosa-sts-advanced-f7/1754678126903824384)